lib/OpenQA/Command.pm

```diff
@@ -89,6 +89,16 @@ sub parse_params {
         push @{$params{$1}}, $2;
     }
 
+    return \%params unless (defined $params{template});
+
+    foreach my $t (@{$params{template}}) {
+        if ($t =~ /^file:(.+)$/) {
+            open(my $fh, '<', $1) || die "ERROR: cannot open file [$1]: $!\n";
+            $t = do { local $/; <$fh> };
+            close $fh;
```

Comment:
It would be nicer to use `Mojo::File::slurp` instead. You could also use `Mojo::URL` for checking whether the schema is `file`.

Reply:
Done.